examples/_generate_example_data.py

```diff
@@ -37,16 +37,15 @@ def generate_example_dataset(name):
         bootstrap replicates, or 'gamma' for gamma mean bootstrap replicates
     :return: Replicates and estimate for the data
     """
-    np.random.seed(10042002)
     rng = np.random.default_rng(seed = 10042002)
     if name == 'normal':
         num_rep, mean, std = 100, 0.74, 0.086
-        raw_data = norm.rvs(size = num_rep, loc = mean, scale = np.sqrt(num_rep) * std)
+        raw_data = rng.normal(size = num_rep, loc = mean, scale = np.sqrt(num_rep) * std)
         replicates = get_replicates(raw_data)
         estimate = np.median(raw_data)
     elif name == 'gamma':
         num_rep, variance, mean = 499, 0.186, 2.189
-        raw_data = gamma.rvs((mean**2)/(variance * num_rep), scale = (variance * num_rep)/mean, size = 50)
+        raw_data = rng.gamma((mean**2)/(variance * num_rep), scale = mean/(variance * num_rep), size = 50)
```

Comment:
@MosesStewart was this change intentional? It has a big effect on the values in the example.

Reply:
[`np.random.seed`](https://numpy.org/doc/stable/reference/random/generated/numpy.random.seed.html) is depreciated, so I updated the code to use `np.random.default_rng()`.

Reply:
@MosesStewart thanks! If you look at the `git diff`, you'll see a big change in the replicate values. I'm guessing that means we implicitly changed the type of gamma we're simulating. I'm guessing that was unintentional?